src/Raven.Server/ServerWide/RawDatabaseRecord.cs

```diff
@@ -960,6 +961,60 @@ public List<GenAiConfiguration> GenAis
             }
         }
 
+        private Dictionary<string, AiAgentConfiguration> _aiAgents;
+
+        public Dictionary<string, AiAgentConfiguration> AiAgents
```

Comment:
I think it will be best to have a list like for other fields (Etls, Replications, etc)?